Source/WebCore/platform/graphics/GraphicsContext.h

```diff
@@ -216,6 +216,10 @@ class GraphicsContext {
     virtual void fillPath(const Path&) = 0;
     virtual void strokePath(const Path&) = 0;
     WEBCORE_EXPORT virtual void drawPath(const Path&);
+#if ENABLE(INLINE_PATH_DATA)
```

Comment:
I hope we could move away from the INLINE_PATH_DATA ifdef, to simplify the interface and correctness testing.